tests/py/dynamo/conversion/test_grid_aten.py

```diff
@@ -126,19 +88,98 @@ class TestGridConverter(DispatchTestCase):
                 grid_sampler_op[1],
                 grid_sampler_op[2],
                 grid_sampler_op[3],
+                grid_sampler_op[4],
+            )
+            for grid_sampler_op in grid_sampler_ops
+        ]
+    )
+    def test_grid(self, _, op_name, op, input_shape, dim_shape):
+        class TestModule(nn.Module):
+            def __init__(self, grid_sampler_op):
+                super().__init__()
+                self.grid_sampler_op = grid_sampler_op
+
+            def forward(self, x):
+                grid = torch.randint(-1, 1, dim_shape, dtype=torch.float32)
+                return self.grid_sampler_op(x, grid, grid_sampler_aten_ops[op_name])
```

Comment:
Would this need to be `grid_sampler_aten_ops[op_name](x, grid)`?

Reply:
@gs-olive yes it could be but then in that case it would be something like-
```
   def test_grid(self, _, op_name, input_shape, dim_shape, padding_mode, interpolation_mode, align_corners):
        class TestModule(nn.Module):
            def __init__(self):
                super().__init__()

            def forward(self, x):
                grid = torch.randint(-1, 1, dim_shape, dtype=torch.float32)
                return grid_sampler_aten_ops[op_name](x, grid, padding_mode, interpolation_mode, align_corners)

        inputs = [torch.randn(input_shape, dtype=torch.float32)]
        grid_model = TestModule()
        self.run_test(grid_model, inputs)
```
It would be code design choice to either  encapsulate it in the lambda function in the parameters, or declare it in the above way. In my opinion both should be good.

You could let me know if you think otherwise.

Reply:
I see - this makes sense, thanks. In that case, it seems good to go - should just need a rebase to enable testing.